Sources/SQLite/Core/Connection.swift

```diff
@@ -119,11 +129,18 @@ public final class Connection {
     ///
     ///     Default: `false`.
     ///
+    ///   - target: An optional target queue for database operations to be executed on.
+    ///     Set this to a custom serial queue if you need to schedule work that must be
+    ///     synchronized with respect to database operations.
+    ///
+    ///     **Important:** The target queue **must not** be a concurrent queue, or access
+    ///     to the returned connection is no longer guaranteed be thread-safe.
```

Comment:
could this be asserted in code?

Reply:
That would be nice! Currently I'm not aware of public API to check whether a queue is serial or concurrent. 🤔